crates/oxc_ast/src/ast_impl/js.rs

```diff
@@ -1,6 +1,3 @@
-// NB: `#[visited_node]` attribute on AST nodes does not do anything to the code in this file.
```

Comment:
Can we keep these comments, but just change `#[visited_node]` to `#[ast]`? The intent is for `#[ast]` to become a real macro in future, but it isn't right now - and anyway when it does we'll need to have a comment here explaining what it does.

Reply:
These are removed from the implementation files where we don't have any occurrence of the `visited_node`(and now `ast`). I can move them to a separate PR if you'd prefer. That's just a cleanup I've missed when separating ast files into x.rs and x_impl.rs

Reply:
Oh I'm sorry, my mistake. I wasn't paying attention to which files these comments were in. No need to put in a separate PR. Just ignore me!